### ISSUE TITLE
Group resource broken on Fedora 22

### ISSUE THREAD

**Issue by joerg** [sometimesfood/wright#15]
The group resource seems to be broken on Fedora 22. You can check out the exact code I used in github.com/joerg/kitchen-wright

Under example is issued the commands:

```
bundle install --path .bundle --gemfile Gemfile.kitchen --binstubs
bin/kitchen converge default-fedora-22
```

which results in:

```
ERROR: group 'ordinary_gentlemen': undefined method `create' for nil:NilClass
/tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/resource/group.rb:39:in `block in create': undefined method `create' for nil:NilClass (NoMethodError)
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/resource.rb:107:in `might_update_resource'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/resource/group.rb:38:in `create'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/resource.rb:72:in `run_action'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/dsl.rb:60:in `yield_resource'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/dsl.rb:45:in `block in register_resource'
    from /tmp/kitchen/lib/wright_include.rb:1:in `<top (required)>'
    from /tmp/kitchen/wright.rb:9:in `require'
    from /tmp/kitchen/wright.rb:9:in `<top (required)>'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/cli.rb:42:in `load'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/cli.rb:42:in `run_script'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/lib/wright/cli.rb:26:in `run'
    from /tmp/kitchen/.bundle/ruby/gems/wright-0.4.2/bin/wright:6:in `<top (required)>'
    from /tmp/kitchen/.bundle/ruby/bin/wright:23:in `load'
    from /tmp/kitchen/.bundle/ruby/bin/wright:23:in `<main>'
```

Debian, Ubuntu and CentOS work just find. Just Fedora seems broken.

**Comment by sometimesfood** [comment]
Thanks for the bug report. This is due to a bug in `Wright::Util.os_family`:

```
% wright -e "p Wright::Util.os_family"
"fedora\nVERSION_ID=22"
```

Distro detection is the most fragile piece of code in the whole codebase. I had hoped to replace it in the next major release, but apparently this needs to be fixed earlier. Oh well.

As a short-term workaround, the [regex that matches ID](https://github.com/sometimesfood/wright/blob/2c412d5b2551f4d7afe9ae2cc5a10ab345479f10/lib/wright/util.rb#L45-L46) should be changed to check for `[^"\n]` instead of `[^"]`. I'll release an updated version tomorrow.

By the way, you did not add `Gemfile.kitchen` to your repo, so the command above does not work.

**Comment by joerg** [comment]
Note to self:. Never push after 10pm... :-)

**Comment by sometimesfood** [comment]
Just for the record: This is fixed in [release v0.4.3](https://github.com/sometimesfood/wright/releases/tag/v0.4.3). I also pushed new Debian packages to [Launchpad](https://launchpad.net/~sometimesfood/+archive/ubuntu/wright).

**Comment by joerg** [comment]
Thanks, tested and works in kitchen-wright's example now.
